### DISCUSSION TITLE
Importing CSV files

### DISCUSSION THREAD

**Discussion by guilhermenovais** [firefly-iii/firefly-iii#8871] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

There seems to be a problem when uploading CSV files on Data Importer since version 1.5.0 (also checked the latest development version). After selecting the file and clicking "Next", the Configuration page is empty. No error is printed on the console. Downgrading to 1.4.5 fixes the issue.

![image](https://github.com/firefly-iii/firefly-iii/assets/61892041/2a248b92-8768-4757-b4d3-171d3406283b)
![image](https://github.com/firefly-iii/firefly-iii/assets/61892041/632fbc60-daff-43a4-94cc-0bc396f8286a)

### Debug information

Debug information generated at 2024-05-09 08:07:14 America/Sao_Paulo for Firefly III Data Importer version **1.5.2**.

<table><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr><tr><td>Version</td><td>1.5.2</td></tr><tr><td>Build</td><td><span>#</span>628, base <span>#</span>83</td></tr><tr><td>System</td><td>PHP 8.3.6, Linux, apache2handler</td></tr></table><table><tr><th colspan="2">App information</th></tr><tr><th>Item</th><th>Value</th></tr><tr><td>Timezone</td><td>America/Sao_Paulo, America/Sao_Paulo</td></tr><tr><td>Environment</td><td>local</td></tr><tr><td>Debug mode</td><td>false, cache 'file'</td></tr><tr><td>Log level</td><td>debug, stack</td></tr><tr><td>Display errors</td><td>Off, ALL errors</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Trusted proxies</td><td>**</td></tr></table><table><tr><th colspan="2">User information</th></tr><tr><th>Item</th><th>Value</th></tr><tr><td>User agent</td><td>Mozilla/5.0 (X11; Linux x86_64; rv:125.0) Gecko/20100101 Firefox/125.0</td></tr></table>

### Expected behaviour

The Configuration page should show the Configuration Form, as it does on version 1.4.5:
![image](https://github.com/firefly-iii/firefly-iii/assets/61892041/bbd5f668-7e77-4c0f-9acd-dd3a3a0a44d3)

### Steps to reproduce

1. Run the latest version of Firefly III Data Importer;
2. Configure the token (I tested both using Personal Access Token and OAuth Clients);
3. On the main page, click "Import file";
4. On Importable file, select a CSV file (test file I'm using: [fireflytest.csv](https://github.com/firefly-iii/firefly-iii/files/15261566/fireflytest.csv));
5. Click "Next".

### Additional info

I'm running a Docker Compose installation running behind a Nginx reverse proxy.

**Comment by JC5** [comment]
What's the error in the browser console when you go to the configuration page?

**Reply by guilhermenovais** [reply]
There is no error in the browser console when I go to the configuration page.

**Reply by JC5** [reply]
Sorry, a basic Docker run reveals a perfectly fine configuration page for me.

**Comment by bitterlox** [comment]
I am experiencing this issue as well, on a self-hosted installation.
edit: had a hunch and successfully resolved the issue by creating my first account in firefly-iii
maybe an error can be added to avoid confusion; I'd do it myself but i don't know the first thing about php
